.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: Testing on Ubuntu
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
```

Comment:
[nits] It's better to specify `timeout-minutes` because CI sometimes gets stuck.